### ISSUE TITLE
Deconz 6.12.1

### ISSUE THREAD

**Issue by brinzlee** [home-assistant/addons#2454]
### Describe the issue you are experiencing

Since upgrading to the latest deconz addon version 6.12.1 I can no longer access the phoscon page. The password dialogue box is no longer visible. I tried to downgrade and restore the previous addon 6.12.0 but this flags an error. So have kept 6.12.1. Deconz still works and I can see the flow of my devices which all appear to work, just Phoscon that has the problem. I have cleared the cache in my browser (chrome) and have also used alternative browsers without resolve. 

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

deCONZ

### What is the version of the add-on?

6.12.1

### Steps to reproduce the issue

1. Just instal; Deconz 6.12.1
2.
3.
...

**Comment by mdegat01** [comment]
I assume this is related to #2325 . Going to take a look. How do you access HA btw? Like the URL in browser, is it an IP address or a domain name? That was what #2325 was supposed to change.

**Comment by mdegat01** [comment]
I installed Deconz 6.12.1, opened it and clicked Phoscon and this is what I see:
![Screen Shot 2022-04-28 at 3 09 12 PM](https://user-images.githubusercontent.com/2037026/165828238-363afc07-d5d9-4693-9cd5-e56f6458a272.png)

It looks like I'm seeing a password box? I am not very familiar with this app as I don't use it personally. So if this isn't the right place please provide more specific steps to reproduce and show the part of the UI with an issue.

**Comment by brinzlee** [comment]
Very strange. I'm using it with a domain name but I have tried with an IP address too..within the framework of HA...if I try and open the page outside of HA framework it works....

**Comment by mdegat01** [comment]
Hm, that is strange. My screenshot was from within the framework of HA btw. That one was from Firefox, just tried on chrome and I see the same. Not entirely sure what to do in that case without being able to reproduce it...

If you open developer tools on your browser is there any errors in the console? The changes with that PR affected the loaded javascript so if it broke something I imagine you would have console errors.

**Comment by brinzlee** [comment]
This is my screen with no password field...I think the issue is my end but not really sure how to resolve it. 

![Capture](https://user-images.githubusercontent.com/14986700/165914780-66427bd8-b3a3-4c9b-a5e8-9a1611637fc6.JPG)

**Comment by brinzlee** [comment]
Uninstalling the APP and reinstalling solved this issue....

**Comment by brinzlee** [comment]
No its irratic.....There is no logic in how sometimes it opens without an issue and other times you can see it entering the password in the field but goes no further.....it won't let you press the login button

**Comment by mdegat01** [comment]
@tannyl Sorry for the ping but since I know you did a deep dive into the deconz js as a part of that recent PR do you have any ideas what could be happening here? Seems like odd new behavior on 6.12.1 and I can't seem to reproduce it.

**Comment by tannyl** [comment]
@mdegat01 No worries. I don't mind helping, so feel free to ping me. :)

@brinzlee I have some questions if you don't mind: What browsers have you tested it in and on what operation systems. And what results did you get in the different browsers and OS's.

The Javascript code (by Dresden Elektronik) used for Phoscon is "wonky". It works, but it is not very pretty or well made (that is my own personal opinion). So I would not be surprised if the cross browser comparability is "wonky" as well.

**Comment by brinzlee** [comment]
Thanks for the help Guys.... @tannyl I have had no problems before the update of deConz to this latest version. I use google chrome as my primary browser and it works when using HA with a duckdns address.....won't work with Nabu Casa address nor and local IP address.....
It works fine with Microsoft Edge with all mediums...Its very odd

**Comment by tannyl** [comment]
Thank you @brinzlee 

TL;DR: It is a bug in the Chrome browser.

**Long version:**
I have looked into it some more. Based on your image from before, I suspected that the input box was actually still there, but for some reason hidden. If the box was completely missing, the "login" button should not still be "floating" to the right like it does.

I have managed to replicate the problem using Chrome (Pop_os / Linux) and my suspicions are correct. The input field is indeed still there.

My conclusion is that this is a bug in Chrome that only triggers on very specific circumstances. I have been able to narrow it down to having `display: inline` and `position: relative` on a deeply nested parent element inside an `iframe`. This results in the element (and any siblings) not being drawn. But the element and any content of it it is still there and works and takes up space as normal. I have not been able to replicate the error in say a jsfiddle.

There are a couple of workarounds:

1. Load the Phoscon page directly (`ctrl + left mouse click` on the blue Phoscon button to open it in a new tab). Then the input element will show up as it should (as well as other elements that are affected).
2. Open the Phoscon page "inside Homeassistant" (in the iframe). Then when at the login page, click on the "empty space" where the input box should be and type in the password. After this the login button will disappear as well. Either press enter or click the "empty space" where the login button used to be.
3. Use a different browser then Chrome.

I do not really suggest using workaround 2, since you will run into missing elements again after you have logged in.

If you did not have this problem before 6.12.1, then I guess Chrome must have been updated about the same time you upgraded the addon, which made it look like it was the addon that had an error, while it is in fact an error in Chrome. :smile:

**Comment by tannyl** [comment]
@brinzlee You are welcome. Maybe this issue should be closed again?
